components/public-api-server/pkg/apiv1/oidc.go

```diff
@@ -81,11 +81,12 @@ func (s *OIDCService) CreateClientConfig(ctx context.Context, req *connect.Reque
 		return nil, status.Errorf(codes.Internal, "Failed to store OIDC client config.")
 	}
 
-	created, err := db.CreateOIDCCLientConfig(ctx, s.dbConn, db.OIDCClientConfig{
+	created, err := db.CreateOIDCClientConfig(ctx, s.dbConn, db.OIDCClientConfig{
 		ID:             uuid.New(),
 		OrganizationID: organizationID,
 		Issuer:         oidcConfig.GetIssuer(),
 		Data:           data,
+		Active:         false,
```

Comment:
This value should come from the incoming request. If I wanted to use the API to automate my OIDC setup, I should be able to mark it active directly without relying on a manual verification flow. But for the normal flow where you set it up through the dashboard, it would be set to false

Reply:
Right. 
Changing it.

Reply:
done